### PR TITLE
Update to Angular 13

### PR DESCRIPTION
**Pull request by KyleMcMaster**
# Description

- update Angular packages to version 13.1
- cleanup Angular.json configuration 
- remove -aot parameter from run dev task
- update a few other packages as well

## Purpose

- [ ] Feature
- [ ] Partial Feature
- [ ] Bugfix
- [x] Docs, Config
- [x] Chore / Format

## Change Type

- [x] Major
- [ ] Minor
- [ ] Fix

## Data Persistence Changes

N/A

## Configuration Changes

N/A

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -3,43 +3,43 @@
...
+    "@angular/platform-browser-dynamic": "13.1.0",
+    "@angular/router": "13.1.0",
+    "@datorama/akita": "7.0.1",
+    "@fortawesome/angular-fontawesome": "0.10.1",
+    "@fortawesome/fontawesome-svg-core": "1.2.36",
+    "@fortawesome/free-solid-svg-icons": "5.15.4",
     "@microsoft/signalr": "3.1.8",
-    "@ng-bootstrap/ng-bootstrap": "7.0.0",
-    "bootstrap": "4.5.2",
+    "@ng-bootstrap/ng-bootstrap": "11.0.0-rc.0",
```
Comment (seangwright) [inline client/package.json:29]: Looks like 11 was fully released
https://github.com/ng-bootstrap/ng-bootstrap/releases/tag/11.0.0

Comment (seangwright) [reply client/package.json:29]: Looks like 12 will support Bootstrap 5 + Angular 13

https://github.com/ng-bootstrap/ng-bootstrap/releases/tag/12.0.0-beta.4

Comment (KyleMcMaster) [reply client/package.json]: updated to release version!

Comment (KyleMcMaster) [reply client/package.json]: i'll have to look into bootstrap 5 and what that offers over v4.

**Review by seangwright — APPROVED** [review]

**Pull request merged**